docs/modules/operation/pages/deep-dive/service-assurance/introduction.adoc

```diff
@@ -2,41 +2,41 @@
-A _Service_ can have any arbitrary name and is associated with a _Service Monitor_.
-For example, we can define two _Services_ with the name _HTTP_ and _HTTP-8080_, both are associated with the _HTTP Service Monitor_ but use a different _TCP port_ configuration parameter.
-The following figure shows how _Pollerd_ interacts with other components in _OpenNMS_ and applications or agents to be monitored.
+This information is based on service monitors which Pollerd stores and runs.
+A service can have any arbitrary name and is associated with a service monitor.
+For example, we can define two services with the name `HTTP` and `HTTP-8080`, both are associated with the HTTP Service Monitor but use a different TCP port configuration parameter.
+The following figure shows how Pollerd interacts with other components in {page-component-title} and applications or agents to be monitored.
 
-The availability is calculated over the last 24 hours and is shown in the _Surveillance Views_, _SLA Categories_ and the _Node Detail Page_.
-Response times are displayed as _Resource Graphs_ of the _IP Interface_ on the _Node Detail Page_.
-Configuration parameters of the _Service Monitor_ can be seen in the _Service Page_ by clicking on the _Service Name_ on the _Node Detail Page_.
-The status of a _Service_ can be _Up_ or _Down_.
+The availability is calculated over the last 24 hours and is shown in the Surveillance Views, SLA Categories, and the Node Detail pages.
+Response times are displayed as resource graphs of the IP interface on the Node Detail page.
+Configuration parameters of the service monitor can be seen in the Service page by clicking on the service name on the Node Detail page.
+The status of a service can be up or down.
 
-TIP: The _Service Page_ also includes timestamps indicating the last time at which the service was polled and found to to be _Up_ (Last Good) or _Down_ (Last Fail).
-These fields can be used to validate that _Pollerd_ is polling the services as expected.
+TIP: The Service page also includes timestamps that indicate the last time at which the service was polled and found to to be Up (last good) or down (last fail).
+These fields help to validate that Pollerd is polling the services as expected.
 
-When a _Service Monitor_ detects an outage, _Pollerd_ sends an _Event_ which is used to create an _Alarm_.
-_Events_ can also be used to generate _Notifications_ for on-call network or server administrators.
-The following images shows the interaction of _Pollerd_ in {page-component-title}.
+When a service monitor detects an outage, Pollerd sends an event. which is used to create an alarm.
+You can also use events to generate notifications for on-call network or server administrators.
+The following images show the interaction of Pollerd in {page-component-title}.
 
-.Service assurance with _Pollerd_ in _OpenNMS_ platform
+.Service assurance with Pollerd in {page-component-title}
```

Comment:
Since the previous line describes the image, I don't think we need the alt text.

```suggestion
```